core/src/main/java/org/apache/any23/writer/RDFWriterTripleHandler.java

```diff
@@ -35,7 +35,7 @@
  */
 public abstract class RDFWriterTripleHandler implements FormatWriter, TripleHandler {
 
-    private final RDFWriter writer;
+    protected final RDFWriter writer;
 
```

Comment:
I am a bit confused why you just did not add setter instead having the filed protected. Or expolit the protected constructor you did below.

Reply:
@jgrzebyta I don't see any reason to disallow protected access to the `writer` if we allow protected access to the constructor. Subclasses could bypass the `private` modifier anyway by simply saving a reference to the writer in the constructor.

Reply:
@HansBrende There is no reason. It is only the coding style.

But if user will override constructor in her/his custom class will call `super(someWriter)` anyway. In that case constructor is a setter for the writer.